--- X:/py-leveldb/versions/download/Python-3.3.0rc2/setup.py	sun. sep.  9 09:11:14 2012
+++ X:/py-leveldb/versions/download/Python-3.3.0rc2/setup.py.new	fös. sep. 21 14:14:55 2012
@@ -518,6 +518,7 @@
             lib_dirs = self.compiler.library_dirs + [
                 '/lib64', '/usr/lib64',
                 '/lib', '/usr/lib',
+                '/usr/lib/x86_64-linux-gnu/'
                 ]
             inc_dirs = self.compiler.include_dirs + ['/usr/include']
         exts = []
